database/DevHome.Database/Migrations/DevHomeDatabaseContextModelSnapshot.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+
+#nullable disable
```

Comment:
Why? Consider adding a comment explaining this and the pragma disable below.

Reply:
This code is auto-generated.